tutorials/tutorials/getting-started.md

```diff
@@ -0,0 +1,55 @@
+# Nevermined SDK Tutorial
+
+## Overview
+
+This tutorial demonstrates a basic example of using the Nevermined SDK to interact with the Nevermined network. The provided code, written in TypeScript, initializes the SDK, configures it, and retrieves information about SDK versions.
+
+## Prerequisites
+
+Ensure you have the following installed:
+
+- Node.js
+- npm (Node Package Manager)
```

Comment:
let's use `yarn` by default since that is what we use internallu

Reply:
done